script.service.hue/resources/lib/qhue/README.md

```diff
@@ -6,15 +6,15 @@ I wrote it because some of the other (excellent) frameworks out there weren't qu
 
 ## Understanding Qhue
```

Comment:
why is this module bundled with the addon and not shared as a script.module? Is it not plain upstream code?

Reply:
I didn't find any other addons that use this module so assumed it wouldn't be accepted, but if this is best practice I can change it

Reply:
Changed as requested, please see associated PR for [script.module.qhue](script.module.qhue).